pkg/network/render.go

```diff
@@ -615,3 +621,15 @@ func renderNetworkDiagnostics(conf *operv1.NetworkSpec, manifestDir string) ([]*
 
 	return manifests, nil
 }
+
+// renderSdnPublic renders the common objects related to the sdn-public namespace
+func renderSdnPublic(manifestDir string) ([]*uns.Unstructured, error) {
+	data := render.MakeRenderData()
+	data.Data["ReleaseVersion"] = os.Getenv("RELEASE_VERSION")
+
+	manifests, err := render.RenderDir(filepath.Join(manifestDir, "network", "sdn-public"), &data)
```

Comment:
Not a fan of the "sdn-public" dir name because of associations with OpenShift SDN when there are other CNIs.